From dd86dd796b8b12c312af62be6135969409ffe0e3 Mon Sep 17 00:00:00 2001
From: Sabeel Syed <sabeel.syed@nxp.com>
Date: Mon, 21 May 2018 17:15:22 +0000
Subject: [PATCH 2/2] Point to proper AWELib models for i.mx7D

The awb, header and Blovk Size differs for i.mx8M and i.mx7D
So, we need to point to the right files regarding the SoC

Signed-off-by: Sabeel Syed <sabeel.syed@nxp.com>
---
 SampleApp/src/PortAudioMicrophoneWrapper.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/SampleApp/src/PortAudioMicrophoneWrapper.cpp b/SampleApp/src/PortAudioMicrophoneWrapper.cpp
index cba9e52..275b582 100644
--- a/SampleApp/src/PortAudioMicrophoneWrapper.cpp
+++ b/SampleApp/src/PortAudioMicrophoneWrapper.cpp
@@ -38,14 +38,14 @@ static const unsigned int NUM_MEDIA_CHANNELS = 2;
 static const unsigned int NUM_AVS_CHANNELS = 1;
 
 
-#include "VoiceUI_imx8_2mic_AVS_v3b_1mb_sensory_AWELibSimple.h"
+#include "VoiceUI_imx7_2mic_AVS_v3_250kb_sensory_AWELibSimple.h"
 static const unsigned int NUM_HARDWARE_INPUT_CHANNELS = 2;
 static const unsigned int NUM_HARDWARE_OUTPUT_CHANNELS = 2;
 static const unsigned int NUM_AWE_INPUT_CHANNELS = 4; // expect an AWE design with 4 input channels
 static const unsigned int NUM_AWE_OUTPUT_CHANNELS = 3; // expect an AWE design with 3 output channels
 static const unsigned int SAMPLE_RATE = 48000; // hardware sample rate
-static const unsigned int BLOCK_SIZE = 768;
-const char awbFilename[] = "/home/root/AWELib/VoiceUI_imx8_2mic_AVS_v3b_1mb_sensory_AWELib.awb";
+static const unsigned int BLOCK_SIZE = 384;
+const char awbFilename[] = "/home/root/AWELib/VoiceUI_imx7_2mic_AVS_v3_250kb_sensory_AWELib.awb";
 
 static const unsigned int AVS_SAMPLE_RATE = 16000;
 static const unsigned int DECIMATION_FACTOR = SAMPLE_RATE/AVS_SAMPLE_RATE; // ratio of hardware sample rate to AVS sample rate
-- 
1.9.1

